tests/integration/llm/prepare.py

```diff
@@ -60,6 +60,11 @@
         "option.s3url": "s3://djl-llm/opt-13b/",
         "option.tensor_parallel_degree": 2,
         "option.task": "text-generation"
+    },
+    "opt-30b": {
```

Comment:
we cannot add this since our testing is on g5 12x. Memory is not enough

Reply:
removed